### PR TITLE
Specify range of Integer type exactly

### PR DESCRIPTION
**Pull request by alanconway**
"32 bit integer" is under-specified, there are multiple ways to encode an
integer in 32 bits with different ranges (signed, unsigned, twos-complement,
ones-complement etc.)

Proposed new wording: "signed 32-bit twos-complement"

### CODE REVIEW

**Comment by duglin** [comment]
@alanconway can you please sign your commit?

File: spec.md [hunk 1]
```diff
@@ -107,7 +107,7 @@ platform/vendor specific protocols (AWS Kinesis, Azure Event Grid).
 
 The following abstract data types are available for use in attributes.
 
-- `Integer` - A 32-bit whole number.
+- `Integer` - A whole number in the range −2,147,483,647 to +2,147,483,647 inclusive. This is the range of a signed 32-bit twos-complement representation. Event formats do not have to use this encoding, but they MUST only use `Integer` values in this range.
```
Comment (duglin) [inline spec.md]: Can you wrap this at 80 columns - so it matches the rest of the doc?

Comment (duglin) [reply spec.md]: https://en.wikipedia.org/wiki/32-bit shows it as "...,648" not "...,647" for the low range.

Comment (alanconway) [reply spec.md]: Thanks, fixed both. All green now :)

Comment (duglin) [reply spec.md]: thanks

**Comment by duglin** [comment]
Approved on the Jan 17 call

**Pull request merged**